binlog_driver/cmd/mysql/mysql.go

```diff
@@ -0,0 +1,266 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/ngaut/log"
+	"github.com/pingcap/tidb-tools/binlog_driver/reader"
+	pb "github.com/pingcap/tidb-tools/binlog_proto/go-binlog"
+)
+
+// a simple example to sync data to mysql
+
+var (
+	port     = flag.Int("P", 3306, "port")
+	user     = flag.String("u", "root", "user")
+	password = flag.String("p", "", "password")
+	host     = flag.String("h", "localhost", "host")
+
+	clusterID = flag.String("clusterID", "6561373978432450126", "clusterID")
+	offset    = flag.Int64("offset", sarama.OffsetNewest, "offset")
+	commitTS  = flag.Int64("commitTS", 0, "commitTS")
+)
+
+func getDB() (db *sql.DB, err error) {
+	dsn := fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?charset=utf8", *user, *password, *host, *port, "test")
+	log.Debug(dsn)
+
+	db, err = sql.Open("mysql", dsn)
+	if err != nil {
+		return
+	}
+
+	return
+}
+
+func main() {
+	flag.Parse()
+
+	cfg := &reader.Config{
+		KafakaAddr: []string{"127.0.0.1:9092"},
+		Offset:     *offset,
+		CommitTS:   *commitTS,
+		ClusterID:  *clusterID,
+	}
+
+	breader, err := reader.NewReader(cfg)
+	if err != nil {
+		panic(err)
+	}
+
+	db, err := getDB()
+	if err != nil {
+		panic(err)
+	}
+
+	for {
+		select {
+		case msg := <-breader.Messages():
+			log.Debug("recv: ", msg.Binlog.String())
+			binlog := msg.Binlog
+			sqls, args := toSQL(binlog)
+
+			tx, err := db.Begin()
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			for i := 0; i < len(sqls); i++ {
+				log.Debug("exec: args: ", sqls[i], args[i])
+				_, err = tx.Exec(sqls[i], args[i]...)
+				if err != nil {
+					log.Error(err)
+				}
+
+			}
+			err = tx.Commit()
+			if err != nil {
+				log.Fatal(err)
+			}
+
+		}
+	}
+
+}
+
+func columnToArg(c *pb.Column) (arg interface{}) {
+	if c.GetIsNull() {
+		return nil
+	}
+
+	if c.Int64Value != nil {
+		return c.GetInt64Value()
+	}
+
+	if c.Uint64Value != nil {
+		return c.GetUint64Value()
+	}
+
+	if c.DoubleValue != nil {
+		return c.GetDoubleValue()
+	}
+
+	if c.BytesValue != nil {
+		return c.GetBytesValue()
+	}
+
+	return c.GetStringValue()
+}
+
+func tableToSQL(table *pb.Table) (sqls []string, sqlArgs [][]interface{}) {
+	replace := func(row *pb.Row) {
+		sql := fmt.Sprintf("replace into `%s`.`%s`", table.GetSchemaName(), table.GetTableName())
+
+		var names []string
+		var holder []string
+		for _, c := range table.GetColumnInfo() {
+			names = append(names, c.GetName())
+			holder = append(holder, "?")
```

Comment:
how about add a function like this 
https://github.com/pingcap/tidb-binlog/blob/4f017856038be57a7acf7a3883bc5526484dc82a/pkg/dml/dml.go#L6

Reply:
it's simple here, let's keep it?